UPGRADE.md

```diff
@@ -2,6 +2,12 @@
 
 ## 3.0.0
 
+### Removing deprecated twig functions
+- `sulu_meta_alternate` (use `sulu_seo` instead)
```

Comment:
`sulu_seo` is deprecated:

Replacement is calling directly the twig file like in the skeleton: https://github.com/sulu/sulu/blob/0c27c41b15a3fc9b6a01387c4451ba78bca76e6a/src/Sulu/Bundle/WebsiteBundle/Twig/Seo/SeoTwigExtension.php#L75-L80

https://github.com/sulu/skeleton/blob/d616fe6b8f8766068a9ae5af84468aa6cd32f5e9/templates/base.html.twig#L9

Reply:
Should I also include the removal of this function in this PR or create a different PR for that?

Reply:
You can remove it in this PR